app/templates/todos/new.plush.html

```diff
@@ -0,0 +1,27 @@
+<%= partial("partials/nav.plush.html") %>
+<div class="container py-5 w-60">
+    <div class="card shadow-sm">
+        <div class="card-body px-5 py-5">
+            <h3 class="card-title font-weight-bold">New Task</h3>
+            <%= form_for(todo, {action: "", method: "POST"}) { %>
+            <div class="form-row pt-4">
+                <div class="col-6">
+                    <%= f.InputTag("Title", {label: "Title", type: "text"}) %>
+                </div>
+                <div class="col-6">
+                    <%= f.InputTag("LimitedDate", {label: "Limit Date", type: "date", value: "", min:"2021-12-12"}) %>
```

Comment:
This should be LimitDate instead of LimitedDate in the model and in this form

Reply:
Done!